app/models/operations/load_service_areas.rb

```diff
@@ -37,10 +41,12 @@ def load_file_data(input)
       issuer_hios_id = sheet.cell(6, 2).to_i.to_s
 
       output = (13..sheet.last_row).each_with_object([]) do |i, result|
+        next unless parse_boolean(sheet.cell(i, 3)).present?
```

Comment:
is this needed? what does this do?

Reply:
Yes @nisanthyaganti9 , parse_boolean(sheet.cell(i, 3)) value is storing into is_all_state key, sometimes this value is nil, am skipping such nil